CHANGELOG.md

```diff
@@ -201,6 +201,7 @@
 ### Chore & Maintenance
 
 - `[website]` Switch domain to https://jestjs.io ([#6549](https://github.com/facebook/jest/pull/6549))
+- `[tests]` Free tests from the dependency on value of FORCE_COLOR ([#6585](https://github.com/facebook/jest/pull/6585/files))
```

Comment:
This is under `23.3.0`, but should be `master`. @hron mind sending a PR fixing the changelog? :)

Reply:
Ooops.. Sorry about that. Here is the [fix](https://github.com/facebook/jest/pull/7234)